docs/config-reference.md

```
@@ -135,6 +309,207 @@ Notes:
+approved_domains = ["example.com"]
+```
+
+Runtime workflow (`web_access_config`):
+
+1. Start strict-first mode (deny unknown domains until reviewed):
+
+```json
+{"action":"set","require_first_visit_approval":true,"enforce_domain_allowlist":false}
+```
+
+2. Dry-run a target URL before access:
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)